crypto/pem/pem_test.cc

```diff
@@ -187,5 +187,81 @@ TEST(PEMTest, WriteReadECPem) {
   const BIGNUM* orig_priv_key = EC_KEY_get0_private_key(ec_key.get());
   const BIGNUM* read_priv_key = EC_KEY_get0_private_key(ec_key_read.get());
   ASSERT_EQ(0, BN_cmp(orig_priv_key, read_priv_key));
+}
+
+const char *kPemECPARAMETERS =
```

Comment:
how did you generate these?

Reply:
`kPemECPARAMETERS` was more easily generate-able. Running the following command on the system's openssl gives you the output.
```
aws-lc on  main [?] via △ v3.20.0 via  v1.21.3 
❯ openssl ecparam -name secp521r1
-----BEGIN EC PARAMETERS-----
BgUrgQQAIw==
-----END EC PARAMETERS-----
```

`kPemExplictECPARAMETERS` was slightly harder, but not too complicated. I took the contents of the explicit P256 curve in [our test file](https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/ec/ec_test.cc#L50-L78) and wrote it out with `PEM_write_bio(..., PEM_STRING_ECPARAMETERS, ...)`.